tests/components/lastfm/__init__.py

```diff
@@ -52,11 +69,15 @@ def get_image(self):
 
     def get_recent_tracks(self, limit):
         """Get mock recent tracks."""
-        return []
+        if self._first_time:
```

Comment:
Feels a bit hacky doing it like this. What about making recent_tracks and top_tracks variables that can be passed in or else default to something. You can then make a fixture for a first time and regular user.